src/components/RefinementList.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div :class="suit('')" v-if="state">
+    <slot
+      :items="items"
+      :refine="state.refine"
+      :searchable="searchable"
+      :search-for-items="state.searchForItems"
+      :show-more="showMore"
+      :toggle-show-more="state.toggleShowMore"
+      :is-showing-more="state.isShowingMore"
+    >
+      <div :class="suit('searchBox')" v-if="searchable">
+        <ais-search-input :refine="state.searchForItems"></ais-search-input>
+      </div>
+      <ul :class="suit('list')">
+        <li
+          :class="[suit('item'), {[suit('item', 'selected')]: item.isRefined}]"
+          v-for="item in items"
+          :key="item.value"
+        >
+          <slot name="item" :item="item" :refine="state.refine">
+            <label :class="suit('label')">
+              <input
+                :class="suit('checkbox')"
+                type="checkbox"
+                :value="item.value"
+                :checked="item.isRefined"
+                @change="state.refine(item.value)"
+              />
+              <template v-if="searchable">
+                <span :class="suit('labelText')" v-html="item.highlighted"></span>
+              </template>
+              <template v-else>
+                <span :class="suit('labelText')">{{item.label}}</span>
+              </template>
+              <span :class="suit('count')">{{item.count}}</span>
+            </label>
+          </slot>
+        </li>
+      </ul>
+      <button
+        :class="suit('showMore')"
```

Comment:
Should have the class `.ais-RefinementList-showMore--disabled` when it's disabled.